cohere/responses/feedback.py

```diff
@@ -1,22 +1,6 @@
-"""Give feedback on a response from the Cohere API to improve the model.
-
-Can be used programmatically like so:
-
-Example: a user accepts a model's suggestion in an assisted writing setting
-```
-generations = co.generate(f"Write me a polite email responding to the one below:\n{email}\n\nResponse:")
-if user_accepted_suggestion:
-    generations[0].feedback(good_response=True)
-```
-
-Example: the user edits the model's suggestion
-```
-generations = co.generate(f"Write me a polite email responding to the one below:\n{email}\n\nResponse:")
-if user_edits_suggestion:
-    generations[0].feedback(good_response=False, desired_response=user_edited_response)
-```
-"""
-
 from typing import NamedTuple
 
 GenerateFeedbackResponse = NamedTuple("GenerateFeedbackResponse", [("id", str)])
+GeneratePreferenceFeedbackResponse = NamedTuple("GeneratePreferenceFeedbackResponse", [("id", str)])
+
+PreferenceRating = NamedTuple("PreferenceRating", [("request_id", str), ("rating", float), ("generation", str)])
```

Comment:
what is request_id here? in tests it seems an id related to the generation rather than the request, but I can't find this anywhere.

Reply:
it's the ID of the co.generate generation